app/Models/Application.php

```
@@ -1555,40 +1555,185 @@ protected function buildGitCheckoutCommand($target): string
+                    $regex .= '\\'.$c;
+                    break;
+
+                default:
+                    $regex .= $c;
+                    break;
+            }
+        }
+
+        // Wrap in delimiters and anchors
+        return '#^'.$regex.'$#';
+    }
```

PHPMD (2.15.0)

The method globToRegex() has a Cyclomatic Complexity of 24. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)